### DISCUSSION TITLE
Can listings dynamically load images so that index pages are more efficient?

### DISCUSSION THREAD

**Discussion by danmackinlay** [quarto-dev/quarto-cli#9434] [Feature Requests]
### Description

over at #9390 we discussed the data costs of quarto listings due to the images they download. there seem to be two issues, i fact. Firstly, there are no thumbnails generated. Secondly listings AFAICT do not lazy load but rather download all media at once, even though most of it will never be viewed. The second fact never got discussed, so I figure it might be better handled as a separate thread here.

So here is my feature idea: can we dynamically load the page images in listing? On one hand, that is potentially slower, if the images are only downloaded on demand. on the other hand, it is already slow; on my website it already takes a long time to download 1000 supporting images for 1000 pages.

**Comment by cscheid** [comment]
PR open here: https://github.com/quarto-dev/quarto-cli/pull/9447
